lib/actions-oclif/scan.ts

```diff
@@ -0,0 +1,165 @@
+/**
+ * @license
+ * Copyright 2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Command, flags } from '@oclif/command';
+import { LocalBalenaOsDevice } from 'balena-sync';
+import { stripIndent } from 'common-tags';
+import * as cf from '../utils/common-flags';
+
+interface FlagsDef {
+	verbose: boolean;
+	timeout?: number;
+	help: void;
```

Comment:
Missing something, why is this void?

Reply:
Good question @hedss !  This is how it's done in the existing oclif commands.  I had meant to look into why, but forgot about it (I had assumed it was an oclif quirk).